src/theme/card-gradient.css.ts

```diff
@@ -54,19 +54,10 @@ export const cardGradientOverlayStyle = recipe({
             background:
               "linear-gradient(180deg, rgba(48, 48, 48, 0.85) 1.22%, rgba(48, 48, 48, 0.0001) 100%)",
             backgroundSize: "cover",
+            zIndex: -1,
           },
           "&:hover:before": {
-            content: " ",
-            display: "block",
-            position: "absolute",
             opacity: "0.9",
-            width: "100%",
-            height: "100%",
-            top: 0,
-            left: 0,
-            background:
-              "linear-gradient(180deg, rgba(48, 48, 48, 0.85) 1.22%, rgba(48, 48, 48, 0.0001) 100%)",
-            backgroundSize: "cover",
```

Comment:
But we need the effect on hover, not always

Reply:
the CSS is the same for hover & not. The difference is the opacity, no?

Reply:
Here is an example from figma
![image](https://user-images.githubusercontent.com/18033717/195808963-35c851b6-a726-4271-a4f3-8b429774892a.png)